### ISSUE TITLE
Items in holograms

### ISSUE THREAD

**Issue by TheWranger** [VolmitSoftware/Gloss#29]
I think it would be cool if we could display floating items in Gloss holograms, like what can be done in HolographicDisplays.

**Comment by cyberpwnn** [comment]
Can you send an image example of it?

Sounds cool, just making sure im giving the armor stand items im assuming?

**Comment by TheWranger** [comment]
No, the items look like actual dropped items. Here's an example from CMI (couldn't quickly find one for HolographicDisplays, but it has this mechanic):
![image](https://user-images.githubusercontent.com/24642838/53788519-30448b00-3f76-11e9-882d-c25aafb9f26e.png)
